test/unit/test.js

```diff
@@ -0,0 +1,10 @@
+var assert = require('chai').assert;
+
+describe('Array', function() {
+  describe('#indexOf()', function () {
+    it('should return -1 when the value is not present', function () {
+      assert.equal(-1, [1,2,3].indexOf(5));
+      assert.equal(-1, [1,2,3].indexOf(0));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://www.codacy.com/app/theprojectshop/Kassy/file/2532315323/issues/source?bid=3282987&fileBranchId=3283883#l7)